src/js/components/FormField/FormField.js

```diff
@@ -245,6 +247,22 @@ const FormField = forwardRef(
       }
     }
 
+    // fileinput handle
+    // use fileinput plain use formfield to drive the border
+    let isFileInputComponent;
+    if (
+      children &&
+      Children.map(children, child => {
+        if (
+          child &&
+          child.type &&
+          FileInputName.indexOf(child.type.displayName) !== -1
+        )
+          isFileInputComponent = true;
+        return child;
```

Comment:
I'm not sure why we need to be returning anything here since it's not getting assigned to any variable. Instead of `map` maybe this should just be a `forEach`? so there is no return